ydb/core/tx/schemeshard/schemeshard_build_index_tx_base.cpp

```diff
@@ -137,19 +139,19 @@ void TSchemeShard::TIndexBuilder::TTxBase::ApplyBill(NTabletFlatExecutor::TTrans
             continue;
         }
 
+        TString id = TStringBuilder()
+            << buildId << "-"
+            << buildInfo.TablePathId.OwnerId << "-" << buildInfo.TablePathId.LocalPathId << "-"
+            << billed.GetUploadRows() + billed.GetReadRows() << "-" << billed.GetUploadBytes() + billed.GetReadBytes() << "-"
+            << processed.GetUploadRows() + processed.GetReadRows() << "-" << processed.GetUploadBytes() + processed.GetReadBytes();
+
         NIceDb::TNiceDb db(txc.DB);
 
-        buildInfo.Billed += toBill;
-        Self->PersistBuildIndexBilling(db, buildInfo);
+        billed += toBill;
+        Self->PersistBuildIndexBilled(db, buildInfo);
 
         ui64 requestUnits = RequestUnits(toBill);
 
-        TString id = TStringBuilder()
-            << buildId << "-"
-            << buildInfo.TablePathId.OwnerId << "-" << buildInfo.TablePathId.LocalPathId << "-"
-            << buildInfo.Billed.GetRows() << "-" << buildInfo.Billed.GetBytes() << "-"
-            << buildInfo.Processed.GetRows() << "-" << buildInfo.Processed.GetBytes();
```

Comment:
Is format of this string fixed? Or it can be any unique string?
If any I suggest using `<upload rows>-<upload bytes>-<read rows>-<read bytes>` instead of  `<upload rows + read rows>-<upload bytes + read bytes>`

Reply:
Ok, I changed it to more detailed format